From: Jan Safranek <jsafrane@redhat.com>
Date: Thu, 9 Jun 2011 14:56:59 +0200
Subject: [PATCH] Load correct shared object name in python.

This fixes following error message shown by wireshark/tshark when wireshark-devel
was not installed:
   libwireshark.so: cannot open shared object file: No such file or directory

diff --git a/epan/wspython/wspy_libws.py b/epan/wspython/wspy_libws.py
index a293a17..0c528e4 100755
--- a/epan/wspython/wspy_libws.py
+++ b/epan/wspython/wspy_libws.py
@@ -35,7 +35,7 @@ def get_libws_libname():
   elif system == "Windows":
     return 'libwireshark.dll'
   else:
-    return 'libwireshark.so'
+    return 'libwireshark.so.3'
 
 def get_libws_handle():
   global __libwireshark
